waiter/src/waiter/handler.clj

```diff
@@ -354,12 +355,33 @@
                           viewable-service-ids)]
       (utils/clj->streaming-json-response response-data))))
 
+(defn- service-ensure-delete-handler-helper
```

Comment:
Nitpick: we start function anmems with verbs when possible. E.g. would prefer this to be renamed to `ensure-service-deleted-locally`